tests/printing/untyped/postfixLambda.flags

```diff
@@ -0,0 +1,2 @@
+-language:experimental.qualifiedTypes
+-language:experimental.postfixLambda
```

Comment:
Could we use a `scalac:` comment in the scala file instead?

Reply:
I did not know this was an option, but all printing tests work like this, so I'd keep it like that, and potentially open an issue/PR to renovate printing tests as a whole